Vagrantfile

```diff
@@ -61,6 +61,8 @@ else
 end
 
 SERVICES_IP = ENV.fetch("NYC_TREES_SERVICES_IP", "33.33.33.30")
+TILER_PORT = ENV.fetch("NYC_TREES_PORT_7000", 7000)
```

Comment:
After some local testing, we should be able to undo most of the changes in here and use the host-only IP address of the tiler VM (without a port) as `tiler_host` instead of `localhost:{{ tiler_port }}`.

Keeping the port forwarding rule seems file to convenience.